service/test/controllers/TelemetryControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package controllers;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.common.io.Files;
+
+import play.mvc.Http.RequestBuilder;
+import play.mvc.Result;
+import play.test.FakeApplication;
+import play.test.Helpers;
+
+/**
+ * 
+ * @author Mahesh Kumar Gangula
+ *
+ */
+
+public class TelemetryControllerTest {
```

Comment:
Only 3 Junits for entire service is very less.

Reply:
5 unit test for controller one method.

Reply:
and 3 unit test for actor